Bumps for 7.0.0-rc2

rmarinho opened the pull request:
### Description of Change

Bumps iOS and Android in same pr

#10286 and #10285

### Issues Fixed

None

jonathanpeppers committed:
Workaround CA1416 repo-wide

Context: https://github.com/dotnet/roslyn-analyzers/issues/6158

jonathanpeppers committed:
Fix for Resizetizer/Catalyst interaction

Context: https://github.com/dotnet/maui/pull/9968#issuecomment-1248103786

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-android build 7.0.1xx-rc2-82e6010400d4949ab54cf8e998e7ad52831c8108-1

Microsoft.Android.Sdk.Windows
 From Version 33.0.0-rc.1.151 -> To Version 33.0.0-rc.2.193

Dependency coherency updates

Microsoft.Dotnet.Sdk.Internal,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net7.Manifest-7.0.100
 From Version 7.0.100-rc.1.22425.9 -> To Version 7.0.100-rc.2.22466.28 (parent: Microsoft.Android.Sdk.Windows

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220922.36

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1247-rc.2-xcode14

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220923.12

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1250-rc.2-xcode14

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220923.13

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1251-rc.2-xcode14

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-bd230eb9-e89b-4dfb-b629-e70cd855a351' into maestro-rc2

rmarinho commented:
/azp run

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220923.30

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1252-rc.2-xcode14

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220923.32

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1253-rc.2-xcode14

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220923.33

Microsoft.iOS.Sdk , Microsoft.MacCatalyst.Sdk , Microsoft.macOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 15.4.1172-rc.2

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220923.31

Microsoft.iOS.Sdk , Microsoft.MacCatalyst.Sdk , Microsoft.macOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 15.4.1171-rc.2

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2

rmarinho committed:
Bump tizen

rmarinho committed:
Fixes for iOS API's

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220926.1

Microsoft.iOS.Sdk , Microsoft.MacCatalyst.Sdk , Microsoft.macOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 15.4.1173-rc.2

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2

rmarinho committed:
Fix other nullable Uri

rmarinho committed:
other fix for null Url

rmarinho committed:
Fix targets for iOS

# Conflicts:
#	.nuspec/Microsoft.Maui.Resizetizer.targets

rmarinho committed:
Move targets to correct place

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220926.7

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1254-rc.2-xcode14

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-android build 7.0.1xx-rc2-a42584443720bc98eab46a61dd372ee17efc0e65-1

Microsoft.Android.Sdk.Windows
 From Version 33.0.0-rc.1.151 -> To Version 33.0.0-rc.2.194

Dependency coherency updates

Microsoft.Dotnet.Sdk.Internal,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net7.Manifest-7.0.100
 From Version 7.0.100-rc.1.22425.9 -> To Version 7.0.100-rc.2.22472.26 (parent: Microsoft.Android.Sdk.Windows

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-bd230eb9-e89b-4dfb-b629-e70cd855a351' into maestro-rc2

rmarinho committed:
Bump required XCODE

mattleibow committed:
Update Microsoft.Maui.Resizetizer.targets

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220927.7

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1255-rc.2-xcode14

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2

rmarinho committed:
Merge branch 'net7.0' into maestro-rc2

# Conflicts:
#	eng/pipelines/common/variables.yml
#	src/Core/src/Platform/iOS/MauiPageControl.cs
#	src/Essentials/src/DeviceDisplay/DeviceDisplay.ios.cs
#	src/Essentials/src/WebAuthenticator/WebAuthenticator.shared.cs

mattleibow committed:
Disable the resizetizer for inner builds of universal iOS/Mac Catalyst apps.

Co-authored-by: Rolf Bjarne Kvinge <rolf@xamarin.com>

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220927.12

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1256-rc.2-xcode14

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220927.10

Microsoft.iOS.Sdk , Microsoft.MacCatalyst.Sdk , Microsoft.macOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 15.4.1174-rc.2

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220927.17

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1257-rc.2-xcode14

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220927.19

Microsoft.iOS.Sdk , Microsoft.MacCatalyst.Sdk , Microsoft.macOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 15.4.1176-rc.2

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-android build 7.0.1xx-rc2-cd05e83c73b1a9a33ec741cda17e9e596087604e-1

Microsoft.Android.Sdk.Windows
 From Version 33.0.0-rc.1.151 -> To Version 33.0.0-rc.2.197

Dependency coherency updates

Microsoft.Dotnet.Sdk.Internal,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net7.Manifest-7.0.100
 From Version 7.0.100-rc.1.22425.9 -> To Version 7.0.100-rc.2.22472.26 (parent: Microsoft.Android.Sdk.Windows

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-android build 7.0.1xx-rc2-16569e7dd221336368d0185eb329a144af58d2bd-1

Microsoft.Android.Sdk.Windows
 From Version 33.0.0-rc.1.151 -> To Version 33.0.0-rc.2.198

Dependency coherency updates

Microsoft.Dotnet.Sdk.Internal,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net7.Manifest-7.0.100
 From Version 7.0.100-rc.1.22425.9 -> To Version 7.0.100-rc.2.22472.26 (parent: Microsoft.Android.Sdk.Windows

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220927.28

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1258-rc.2-xcode14

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-android build 7.0.1xx-rc2-edc4ba97c7f79cd135a4a63d6cac39c6a09dd130-1

Microsoft.Android.Sdk.Windows
 From Version 33.0.0-rc.1.151 -> To Version 33.0.0-rc.2.199

Dependency coherency updates

Microsoft.Dotnet.Sdk.Internal,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net7.Manifest-7.0.100
 From Version 7.0.100-rc.1.22425.9 -> To Version 7.0.100-rc.2.22472.26 (parent: Microsoft.Android.Sdk.Windows

mattleibow committed:
Merge branch 'release/7.0.1xx-rc2' into maestro-rc2

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-android build 7.0.1xx-rc2-27c554e59ccda7092f64cf05bd03d7cb1283946f-1

Microsoft.Android.Sdk.Windows
 From Version 33.0.0-rc.1.151 -> To Version 33.0.0-rc.2.200

Dependency coherency updates

Microsoft.Dotnet.Sdk.Internal,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net7.Manifest-7.0.100
 From Version 7.0.100-rc.1.22425.9 -> To Version 7.0.100-rc.2.22472.26 (parent: Microsoft.Android.Sdk.Windows

mattleibow committed:
Not yet iOS 16, but will always use the new API

mattleibow committed:
Merge branch 'release/7.0.1xx-rc2' into maestro-rc2

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-bd230eb9-e89b-4dfb-b629-e70cd855a351' into maestro-rc2

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2

rmarinho committed:
Revert "Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2"

This reverts commit 43fa1065bf2bee7468e19ac958128c4a9abc6341, reversing
changes made to d8d8cfb49315f2bbc3e121dedc0bd54b85f24faa.

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220927.31

Microsoft.iOS.Sdk , Microsoft.MacCatalyst.Sdk , Microsoft.macOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 15.4.1177-rc.2

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220928.7

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1259-rc.2-xcode14

rmarinho committed:
Revert "Revert "Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2""

This reverts commit 88f73732af20789eaff7247203e7c413442a76d2.

rmarinho committed:
Add missing feeds for 6.0.10

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2

rmarinho committed:
Add DisableTransitiveFrameworkReferenceDownloads for wpf/winforms packages not being restored

rmarinho committed:
[Android] remove uses-sdk

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220928.18

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1260-rc.2-xcode14

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220928.19

Microsoft.iOS.Sdk , Microsoft.MacCatalyst.Sdk , Microsoft.macOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 15.4.1179-rc.2

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220928.14

Microsoft.iOS.Sdk , Microsoft.MacCatalyst.Sdk , Microsoft.macOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 15.4.1178-rc.2

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-android build 7.0.1xx-rc2-4ac2ae3d2a218c5539b2ab1425f7678b5db1a478-1

Microsoft.Android.Sdk.Windows
 From Version 33.0.0-rc.1.151 -> To Version 33.0.0-rc.2.201

Dependency coherency updates

Microsoft.Dotnet.Sdk.Internal,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net7.Manifest-7.0.100
 From Version 7.0.100-rc.1.22425.9 -> To Version 7.0.100-rc.2.22472.26 (parent: Microsoft.Android.Sdk.Windows

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-android build 7.0.1xx-rc2-bf10ce5525d97f2303993d631003df54f2837617-1

Microsoft.Android.Sdk.Windows
 From Version 33.0.0-rc.1.151 -> To Version 33.0.0-rc.2.202

Dependency coherency updates

Microsoft.Dotnet.Sdk.Internal,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.net7.Manifest-7.0.100
 From Version 7.0.100-rc.1.22425.9 -> To Version 7.0.100-rc.2.22477.20 (parent: Microsoft.Android.Sdk.Windows

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220928.34

Microsoft.iOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 16.0.1261-rc.2-xcode14

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-macios build 20220928.33

Microsoft.iOS.Sdk , Microsoft.MacCatalyst.Sdk , Microsoft.macOS.Sdk , Microsoft.tvOS.Sdk
 From Version 15.4.1029-rc.1 -> To Version 15.4.1180-rc.2

rmarinho committed:
Fix csproj

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-1c7d3b69-cff6-41c2-bb77-7222ce464abc' into maestro-rc2

rmarinho committed:
Merge branch 'darc-release/7.0.1xx-rc2-bd230eb9-e89b-4dfb-b629-e70cd855a351' into maestro-rc2

rmarinho committed:
Try again

jonathanpeppers reviewed (commented):
I think you need `-p:DisableTransitiveFrameworkReferenceDownloads=true` somewhere passed to the template tests in cake:
```
error NU1102: Unable to find package Microsoft.WindowsDesktop.App.Ref with version (= 6.0.10)
error NU1102: Unable to find package Microsoft.AspNetCore.App.Ref with version (= 6.0.10)
```

rmarinho committed:
Remove uses-sdk

rmarinho committed:
DisableTransitiveFrameworkReferenceDownloads

rmarinho committed:
Don't try to BuildInParallel the sample app

rmarinho committed:
Remove PublishTrimmed on Benchmark project

jonathanpeppers committed:
Revert "Remove PublishTrimmed on Benchmark project"

This reverts commit 0b2b7a21b3ca552865142205506cd23de1ca5e2d.

jonathanpeppers committed:
Disable AOT for Benchmarks.Droid.csproj

I might be the only one that ever uses this project, but I believe we opted into AOT the default for Android and this project shouldn't use it.

jonathanpeppers committed:
Set sandbox sample to a single catalyst RID

jonathanpeppers committed:
Revert "Don't try to BuildInParallel the sample app"

This reverts commit e7e9ea53d6dfd85326045d482d2e80c0cda46389.

jonathanpeppers committed:
Update MauiRazorClassLibrarySample.csproj

Try multi-targeting this project

jonathanpeppers committed:
Revert "Update MauiRazorClassLibrarySample.csproj"

This reverts commit 765d0b4db1569e573d9e9669e36e03b6d7e85e10.

rmarinho committed:
Allow set MaxCpuCount

rmarinho committed:
Merge branch 'maestro-rc2' of https://github.com/dotnet/maui into maestro-rc2

jonathanpeppers committed:
Revert "Set sandbox sample to a single catalyst RID"

This reverts commit e0ba479abde8fc714c78290466085f8be0775a0e.

jonathanpeppers committed:
Samples also need `-maxCpuCount: 1`

Plus some C# formatting
